airflow/providers/dbt/cloud/operators/dbt.py

```diff
@@ -212,7 +214,7 @@ def execute(self, context: Context) -> None:
         response = hook.get_job_run_artifact(
             run_id=self.run_id, path=self.path, account_id=self.account_id, step=self.step
         )
-
+        Path(self.output_file_name).parent.mkdir(parents=True, exist_ok=True)
```

Comment:
If we’re using pathlib, it’d be a good idea to also rewrite the code below to reuse the same Path object instead of throwing it away.

Reply:
yes.